plugins/notifiers/slack/client.go

```diff
@@ -137,23 +142,23 @@ func (n *notifier) sendRequest(req *http.Request) (*userResponse, error) {
 }
 
 func parseMessage(message domain.NotificationMessage, templates domain.NotificationMessages) (string, error) {
-	var text string
+	var block string
 	switch message.Type {
 	case domain.NotificationTypeAccessRevoked:
-		text = templates.AccessRevoked
+		block = templates.AccessRevoked
 	case domain.NotificationTypeAppealApproved:
-		text = templates.AppealApproved
+		block = templates.AppealApproved
 	case domain.NotificationTypeAppealRejected:
-		text = templates.AppealRejected
+		block = templates.AppealRejected
 	case domain.NotificationTypeApproverNotification:
-		text = templates.ApproverNotification
+		block = templates.ApproverNotification
 	case domain.NotificationTypeExpirationReminder:
-		text = templates.ExpirationReminder
+		block = templates.ExpirationReminder
 	case domain.NotificationTypeOnBehalfAppealApproved:
-		text = templates.OthersAppealApproved
+		block = templates.OthersAppealApproved
 	}
```

Comment:
hi, just have some thought, to improve readability rather than use switch how about to use map instead?

```go
	templateMap := map[string]string{
		domain.NotificationTypeAccessRevoked:          templates.AccessRevoked,
		...
	}

	text, ok := templateMap[message.Type]
	if !ok {
		return "", fmt.Errorf("template not found for message type %s", message.Type)
	}
```

Reply:
Done